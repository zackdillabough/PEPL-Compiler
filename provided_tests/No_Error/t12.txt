POLY F1(X) = X + 1;
POLY F2 = x^2 + x + 1;
POLY F3(R, T) = R - T;
START
INPUT X;
F3(1, X);
F1(X);
F2(5);
34 21 4
