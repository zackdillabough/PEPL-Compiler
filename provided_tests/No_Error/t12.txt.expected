-33
35
31
